src/pages/Schedule.js

```diff
@@ -0,0 +1,314 @@
+                            3. Food & Agriculture <br/>
+                            4. Wildcard <br/>
+                            ​<br/> 
+                            <b>High School Tracks</b><br/>
+                            1. Spark Teen High School Track <br/></Text><br/>
+                    <Button>Find Your Track</Button><br/>
+                </EventNoDate>
+                <EventWithDate>
+                <Subtitle fontSize="0.6rem">Individual Registration</Subtitle>
+                <EventWithDateInfo date="MAR 29 11:59 PM EST"
+                                description={['Register individually and start building your team through our online PCV Prize Discord community and stay up to date with our latest news.']}
+                                btext="Register Now!"
+                                blink="https://tinyurl.com/PCVPrizeS21"/> 
+                </EventWithDate>
+                <EventWithDate>
+                <Subtitle fontSize="0.6rem">Team Registration</Subtitle>
+                    <EventWithDateInfo date="APR 5 11:59 PM EST"
+                                        description={['Teams will need to fully register by the deadline to qualify for the competition. Teams will need to choose a track in which to complete. Full rules can be found', 
+                                                        'here https://www.prize.pennclimateventures.org/official-rules', '.']}
+                                        btext="Team Registration"
+                                        blink="https://tinyurl.com/PCVPrizeTeamRegistration"/>
+                </EventWithDate>
+            </EventGroup>
+            <EventGroup>
+                <Subtitle fontColor={GREEN}>Attend a State-of-the-Field Panel</Subtitle>
+                <Text>Discussions about current research, innovations, and challenges with leading experts in the fields.</Text>
+                <EventWithDate> 
+                    <Subtitle fontSize="0.6rem">Waste & Circularity</Subtitle>
+                    <Text fontSize="0.2rem">Co-sponsored by 
+                    <Link href="https://www.sustainability.upenn.edu/partners/student-groups"> Penn SSAP</Link></Text>                
+                    <EventWithDateInfo date="MAR 30 8 PM EST"
+                                    description={['Dr. Jim Hagan @' ,'Penn LPS https://earth.sas.upenn.edu/people/james-hagan', 
+                                                    '\n Shuo Yang @', 'Fifty Years https://fiftyyears.com/team/shuo-yang', 
+                                                    '\n with Caitlyn McCloskey from PEG as moderator']}/>
+                </EventWithDate>
+
+                <EventWithDate>
+                    <Subtitle fontSize="0.6rem">Energy & Transportation</Subtitle>
+                    <Text fontSize="0.2rem">Co-sponsored by the
+                    <Link href="http://pennenvironmentalgroup.weebly.com/"> Penn Environmental Group</Link></Text>                
```

Comment:
The leading whitespace in Penn Environmental Group will be removed when the JSX is rendered, so remove it here.